src/platform_impl/linux/x11/util/hint.rs

```diff
@@ -317,13 +317,13 @@ impl XConnection {
 
     pub fn get_normal_hints(&self, window: ffi::Window) -> Result<NormalHints<'_>, XError> {
         let size_hints = self.alloc_size_hints();
-        let mut supplied_by_user: c_long = unsafe { mem::uninitialized() };
+        let mut supplied_by_user = MaybeUninit::uninit();
```

Comment:
This is just an integer. There is no need for uninit.

You can just set it normally (and change the param back to `&mut supplied_by_user` below:

```rust
let mut supplied_by_user = 0;
```

Reply:
The value is ignored. There's no `unsafe`. There's no undefined behavior.

Why zero-initialize a variable that is never read?